core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -386,9 +386,6 @@ final case class KubernetesServiceKindName(value: String) extends AnyVal
 final case class KubernetesRuntimeConfig(numNodes: NumNodes, machineType: MachineTypeName, autoscalingEnabled: Boolean)
 final case class NumNodepools(value: Int) extends AnyVal
 
-//used in pubsub messaging to indicate the cluster and dummy nodepool to be created
-final case class CreateCluster(clusterId: KubernetesClusterLeoId, defaultNodepoolId: NodepoolLeoId)
```

Comment:
I found there wasn't much use for this product type now that we have `ClusterNodepoolAction`